### ISSUE TITLE
In-browser notifications 

### ISSUE THREAD

**Issue by obla** [ozfortress/citadel#269]
Would this be a good idea?

**Comment by BenjaminSchaaf** [comment]
If you're referring to this: https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API, I think its a good idea. But would take some work to get it functioning properly.

**Comment by obla** [comment]
Yes that is what I'm referring to. Not priority. Would help with speeding up match comms.
